Firmware/ultralcd.cpp

```diff
@@ -5193,7 +5203,9 @@ static void lcd_main_menu()
     } else if (!Stopped) {
         MENU_ITEM_SUBMENU_P(_i("Preheat"), lcd_preheat_menu);////MSG_PREHEAT c=18
     }
-
+    if (printer_status < PrinterStatus::IsSDPrinting) {
```

Comment:
Shouldn't this be `printer_status != PrinterStatus::IsSDPrinting`? I think that will read more clearly what the intention is with the code.

Reply:
Also how does `M79` come into this now that we have it?

Reply:
> Also how does M79 come into this now that we have it?

The LCD menu Set Ready [Yes|No]  is only shown when the M79 host timer is running